runtime/centrifuge/src/lib.rs

```diff
@@ -19,13 +19,16 @@
 // Allow things like `1 * CFG`
 #![allow(clippy::identity_op)]
 
+// TODO(cdamian): Use V3?
+use ::xcm::latest::{MultiAsset, MultiLocation};
```

Comment:
Should we use v3 here as per the update that we have planned?

Reply:
Yes. I would go away from using latest anywhere if possible. Then we recognize breaking changes.